web/package.json

```diff
@@ -10,7 +10,7 @@
     "scaffold": "mkdir -p dist && cp src/index.html dist/ && cp ../Staticfile dist/ && cp -r src/static dist/",
     "build": "npm run scaffold && NODE_ENV=production webpack -p",
     "lint": "eslint 'src/**/*.js'",
-    "start": "webpack-dev-server --host 0.0.0.0 --port 8001 --history-api-fallback --content-base src/",
+    "start": "webpack-dev-server --config webpack.config.dev.js",
```

Comment:
Moved a bunch of this config stuff into the config file, because having it spread around doesn't make a ton of sense.